index.bs

```diff
@@ -89,30 +112,13 @@ interface VRDisplay : EventTarget {
    */
   readonly attribute DOMString displayName;
 
-  /**
-   * Populates the passed VRFrameData with the information required to render
-   * the current frame.
-   */
-  boolean getFrameData(VRFrameData frameData);
+  VRAttachedFrameOfReference createAttachedFrameOfReference();
```

Comment:
Yeah, I hear ya.  The concern I have with saying 3DOF is that there are positional elements when neck modeling is included so it's not *actually* 3DOF.  Open to discussion!